frontend/app/utils/string-utils.ts

```diff
@@ -35,3 +35,12 @@ export const padWithZero = (value: number, maxLength: number) => {
   if (value.toString().length >= maxLength) return value.toString();
   return value.toString().padStart(maxLength, '0');
 };
+
+/**
+ *
+ * @param s - the string to be converted to sentence case
+ * @returns - the string with the first letter capitalized and the rest in lowercase
+ */
+export function toSentenceCase(s: string): string {
```

Comment:
I believe we won't be needing this function. We can conveniently capitalize the first letter of a sentence using the Tailwind CSS class `first-letter:uppercase` applied to a block element. I suggest refraining from modifying data, except for specific cases like SIN or Postal Code, solely for display purposes. Achieving the desired outcome with CSS seems more suitable in this scenario.